### DISCUSSION TITLE
Function get_magic_quotes_gpc() is deprecated

### DISCUSSION THREAD

**Discussion by NVThanh96** [mpdf/mpdf#1861] [Q&A]
Function get_magic_quotes_gpc() is deprecated in C:\wamp\www\Joomla3\libraries\cms\application\administrator.php on line 119

1 | 0.0001 | 357512 | {main}( ) | ...\index.php:0
2 | 0.0135 | 596728 | JApplicationCms->execute( ) | ...\index.php:51
3 | 0.0135 | 596728 | JApplicationAdministrator->doExecute( ) | ...\cms.php:257

i use joomla 3.5 and wampp 3.3.0
i have just install and login 
it show some notice

**Comment by finwe** [comment]
This is not related to mPDF.
